dnSpy/dnSpy.Decompiler/DecompilerBase.cs

```
@@ -245,6 +246,9 @@ public abstract class DecompilerBase : IDecompiler {
 		protected static string? GetRuntimeDisplayName(ModuleDef module) => TargetFrameworkInfo.Create(module).ToString();
 		public virtual bool CanDecompile(DecompilationType decompilationType) => false;
 
-		public virtual void Decompile(DecompilationType decompilationType, object data) => throw new NotImplementedException();
+		public virtual void Decompile(DecompilationType decompilationType, object data) => throw new NotImplementedException();
+
+		public virtual void WriteRegionBegin(IDecompilerOutput output, string text) => throw new NotImplementedException();

```

Why throw instead of just ignoring it?